charger/prachtalpha.go

```diff
@@ -36,8 +36,9 @@ type PrachtAlpha struct {
 }
 
 const (
-	prachtMaxCurrent = 40004 - 40001
-	prachtStatus     = 30107 - 30001
+	prachtTotalCurrent = 40004
+	prachtMaxCurrent   = 40003 - 40001
+	prachtStatus       = 30107 - 30001
```

Comment:
```suggestion
	prachtTotalCurrent = 40003 - 40001 // total limit of all connectors
	prachtMaxCurrent   = 40004 - 40001 // +1 for second connector
	prachtStatus       = 30107 - 30001 // +1 for second connector
```

Irgendwas ist hier durcheinander geraten